manifests/web.pp

```diff
@@ -56,4 +56,5 @@
   ip   => $interface ? { /\S+/ => $::networking['interfaces'][$interface]['ip'], default => $::ipaddress },
   port => $apache_port,
   host => $::hostname,
+  status_codes => 302,
```

Comment:
should be [200,302] otherwise do it doesn't fail once the user configures wordpress.